policies/aws.route53.-.connectiontime.json

```diff
@@ -0,0 +1,27 @@
+{
+  "policy": {
+    "actions": [
+      {
+        "category": 2,
+        "type": "event"
+      }
+    ],
+    "conditions": [
+      {
+        "analytic": "baselineDeviation",
+        "operator": ">",
```

Comment:
should any level be here ?

Reply:
level isn't necessary for band deviations